src/main/java/com/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,120 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.ProductDao;
+import com.internet.shop.exceptions.DataProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    private static final String INSERT_SQL =
+            "INSERT INTO products (productName, price) VALUES (?, ?)";
+    private static final String UPDATE_SQL =
+            "UPDATE products SET productName =?,price=? WHERE product_id = ?";
+    private static final String SELECT_ALL_SQL =
+            "SELECT * FROM products WHERE is_deleted = 0";
+    private static final String SELECT_ONE_SQL =
+            "SELECT * FROM products WHERE product_id = ? AND is_deleted = 0";
```

Comment:
```suggestion
            "SELECT * FROM products WHERE product_id = ? AND is_deleted = FALSE";
```
same with other boolean queries

Reply:
done